server.js

```diff
@@ -1,33 +1,160 @@
-import express from 'express'
-import bodyParser from 'body-parser'
-import cors from 'cors'
-
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
-
-// Defines the port the app will run on. Defaults to 8080, but can be 
+import express from 'express';
+import bodyParser from 'body-parser';
+import cors from 'cors';
+
+import booksData from './data/books.json';
+
+// Defines the port the app will run on. Defaults to 8080, but can be
 // overridden when starting the server. For example:
 //
 //   PORT=9000 npm start
-const port = process.env.PORT || 8080
-const app = express()
+const port = process.env.PORT || 8080;
+const app = express();
 
 // Add middlewares to enable cors and json body parsing
-app.use(cors())
-app.use(bodyParser.json())
+app.use(cors());
+app.use(bodyParser.json());
+
+const myEndpoints = require('express-list-endpoints');
 
-// Start defining your routes here
+// / endpoint (Root - Homepage)
+// RETURNS: A list of available endpoints (in an array)
 app.get('/', (req, res) => {
```

Comment:
Hmm this endpoint doesn't make much sense. I get what you wanted to achieve, but you should rather use `app.use()` method and combine it with `next()` method to detect if something went wrong. Moreover, 404 error should be used only if we didn't find records which fulfill our query (For example when we queried for a book with title "Technigo guides", but there was no book like this in database). In your case, I would rather suggest 500 or 503